pkg/client/client.go

```diff
@@ -11,7 +11,19 @@ import (
 	"github.com/aep-dev/aep-lib-go/pkg/api"
 )
 
-func Create(ctx context.Context, r *api.Resource, c *http.Client, serverUrl string, body map[string]interface{}, parameters map[string]string) (map[string]interface{}, error) {
+type Client struct {
+	Headers map[string]string
+	c       *http.Client
```

Comment:
I don't feel strongly about this, but this kind of overloads what the "c" variable means. I feel like fields in a struct should be fully typed out (`client`)? not only does that help with the confusion, but also makes it easier to a reader.

Reply:
Yeah, I'll rename it. I'm not a big fan of Go's single-letter variable names (that's supposed to be a way to denote private variables? whatever...)